src/tests/quantuminspire/qiskit/test_backend_qx.py

```diff
@@ -163,10 +165,10 @@ def test_get_experiment_results_raises_simulation_error_when_no_histogram(self):
         job.job_id.return_value = '42'
         simulator = QuantumInspireBackend(api, Mock())
         with self.assertRaises(QisKitBackendError) as error:
-            simulator.get_experiment_results(job)
+            simulator.get_experiment_results(job, False)
```

Comment:
The meaning of the second argument is not clear. Consider using the explicit parameter name: `only_latest_run=False`

Reply:
Done for all the instances